java/clients/src/main/java/sleeper/status/report/job/StandardProcessRunReporter.java

```diff
@@ -119,6 +121,21 @@ public static String formatDecimal(double value) {
         return decimalWithCommas("%.2f", value);
     }
 
+    public static String formatDurationStringSeconds(double seconds) {
+        return formatDurationStringMillis(seconds * 1000);
+    }
+
+    public static String formatDurationStringMillis(double millis) {
+        return formatDurationString(Duration.ofMillis((long) millis));
```

Comment:
The parameter should probably be a long given it's immediately cast to that. This could also be private, or maybe merged with the function above.